ambari-logsearch/ambari-logsearch-web/src/app/services/user-settings.service.ts

```diff
@@ -87,7 +115,7 @@ export class UserSettingsService {
           filter: newConfig
         }, null, {
           clusterName
-        });
+        }).first().subscribe(addResponseHandler(clusterName), addResponseHandler(clusterName));
```

Comment:
isn't `first()` redundant here, since it's already added to `request` method override in `HttpClientService`?